### PR TITLE
Ingest SCiL 2020 (fix)

### PR DESCRIPTION
**Pull request by mjpost**
Last week's ingestion of SCiL (#984) contained the submission PDFs, instead of the camera-readies. This fixes that.

### CODE REVIEW

**Comment by akoehn** [comment]
Is there a reason for the deleted papers? They seem to be on the scil publication list: https://scholarworks.umass.edu/scil/vol3/iss1/

**Comment by mjpost** [comment]
I've pinged the organizers for comment and hope they will comment here.

**Comment by MaxAndrewNelson** [comment]
We did not have the camera-ready versions submitted through Easy Chair, which is the cause for the original discrepancy. I had to manually assign paper numbers to the camera-ready submissions so that they would match the numbers given in Easy Chair, in doing so I noticed that submission 92, 93, and 94 were present twice in the Easy Chair files (duplicated in 86, 52, and 60 respectively) and so I removed one copy from the list.

**Comment by mjpost** [comment]
@MaxAndrewNelson to be sure we get this right, can you confirm that the [currently-ingested entries](https://www.aclweb.org/anthology/volumes/2020.scil-1/) is correct, and this submission is just a reordering? Ideally, you would submit this draft in the same order as currently published, so that we don't break any assigned IDs, but are only changed the PDFs and one piece of the metadata (the page number assignments).

**Comment by MaxAndrewNelson** [comment]
I can confirm that those entries are correct. All that is changed is the PDFs, however rather than a reordering this includes the removal of 3 duplicate entries, which were the last three entries in the original submission. Because those 3 entries were all at the end, the remaining submissions are in the same order as the currently published version. However the page numbers will still be changed because camera-ready versions may have a different length than the original submission.

**Comment by mjpost** [comment]
Thanks. But:

* On [this page](https://www.aclweb.org/anthology/volumes/2020.scil-1/) (the currently hosted version in the Anthology), there are no duplicates. 
* If you look at [the diff](https://github.com/acl-org/acl-anthology/pull/994/files), though, there are no redundant papers removed—just a reordering.

Can you resubmit the proceedings to me, retaining the original paper order as delivered to the Anthology? In general, once an ID has been assigned, we do not change it. So if we can restore the original ordering that was submitted to the Anthology, the fix is pretty straightforward.

**Comment by MaxAndrewNelson** [comment]
Sorry about that, I see the problem now. A .tgz can be found [at this link](https://drive.google.com/drive/folders/17ozKHIp5i97mCwuzvnahmv8847bnXhcY?usp=sharing). This will have the camera-ready versions in the same order as my original submission. All PDFs are correct, page numbers will differ only because lengths of documents may have changed, there is no reordering with respect to the first submission.

**Comment by mjpost** [comment]
Thank you! This mostly looks good, except for the following title?

> +      <title>Automatic Induction of Minimalist grammars5(5),5(5),5(5)</title>

**Comment by MaxAndrewNelson** [comment]
Oops a tabs/spaces mixup, updated at the same link.

**Comment by mjpost** [comment]
Now the file has the incorrect order.

**Comment by mjpost** [comment]
Correction! I did the `diff` against the wrong branch. This last file looks good. Thanks, @MaxAndrewNelson.

**Comment by mjpost** [comment]
@akoehn do you have time to look this over?

File: data/xml/2020.scil.xml [hunk 1]
```diff
@@ -18,292 +18,292 @@
...
       <title>What Don’t <fixed-case>RNN</fixed-case> Language Models Learn About Filler-Gap Dependencies?</title>
       <author><first>Rui</first><last>Chaves</last></author>
-      <pages>1–10</pages>
-      <url hash="011c5af8">2020.scil-1.1</url>
+      <pages>1–11</pages>
+      <url hash="4d706147">2020.scil-1.1</url>
     </paper>
     <paper id="2">
       <title>Assessing the ability of Transformer-based Neural Models to represent structurally unbounded dependencies</title>
       <author><first>Jillian Da</first><last>Costa</last></author>
```
Comment (danielgildea) [inline data/xml/2020.scil.xml:26]: "Da Costa"

File: data/xml/2020.scil.xml [hunk 2]
```diff
@@ -18,292 +18,292 @@
...
       <author><first>Natalia</first><last>Talmina</last></author>
       <author><first>Tal</first><last>Linzen</last></author>
-      <pages>42–50</pages>
-      <url hash="2f4b30aa">2020.scil-1.6</url>
+      <pages>49–58</pages>
+      <url hash="d1069898">2020.scil-1.6</url>
     </paper>
     <paper id="7">
       <title><fixed-case>MG</fixed-case> Parsing as a Model of Gradient Acceptability in Syntactic Islands</title>
       <author><first>Aniello De</first><last>Santo</last></author>
```
Comment (danielgildea) [inline data/xml/2020.scil.xml:62]: "De Santo"

**Review by danielgildea — APPROVED** [review]

**Pull request merged**